src/monodroid/jni/debug.hh

```diff
@@ -2,7 +2,7 @@
 #ifndef __MONODROID_DEBUG_H__
 #define __MONODROID_DEBUG_H__
 
-#include <stdint.h>
+#include <cstdint>
 #include <pthread.h>
 
```

Comment:
Are you missing a `<sys/time.h>` include here?

```
12:15:41  "/Users/builder/jenkins/workspace/xamarin-android-pr-pipeline-release/xamarin-android/src/monodroid/monodroid.csproj" (default target) (33) ->
12:15:41  (_BuildHostRuntimes target) -> 
12:15:41    ../../../jni/debug.hh(75,3): error GF2D19E48: unknown type name 'timeval' [/Users/builder/jenkins/workspace/xamarin-android-pr-pipeline-release/xamarin-android/src/monodroid/monodroid.csproj]```

Reply:
Huh, interesting. I didn't see that locally.